baker/algolia/indexExplorersToAlgolia.ts

```diff
@@ -120,17 +121,22 @@ const getExplorerRecords = async (
 
     // Fetch info about all charts used in explorers, as linked by the explorer_charts table
     const graphersUsedInExplorers = await db
-        .knexRaw<{ chartId: number }>(
+        .knexRaw<DbInsertChart>(
```

Comment:
This should be `DbRawChart`. Our nomenclature for the types for DB queries:
**DbInsertENTITY**: only fields are mandatory that must be present in an insert query (e.g. id is optional). This is useful for inserts, but for selects we are guaranteed to have all fields present.
**DbRawENTITY**: all fields are required (because all will be present in a query that queries all columns) but fields that need special parsing are unparsed (e.g. json fields are still strings)
**DbEnrichedENTIYT**: all fields are required and fields requiring special parsing have been parsed.

Reply:
Ah, thanks for the helpful context! I fixed this and the other suggestion now 🙌